### PR TITLE
FF128Relnote: RTCRtpTransceiver.setCodecPreferences() supported

### PR DESCRIPTION
**Pull request by hamishwillee**
FF128 supports [`RTCRtpTransceiver.setCodecPreferences()`](https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpTransceiver/setCodecPreferences) in https://bugzilla.mozilla.org/show_bug.cgi?id=1396922

This adds a release note.

Related docs work can be tracked in #33981

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/128/index.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ This article provides information about the changes in Firefox 128 that affect d
 
 ### APIs
 
+- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preference order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/releases/128/index.md:41]:
```suggestion
- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preference order. Web applications can use this to cause the remote peer to choose a preferred codec — including those used for retransmission, redundancy, and forward error correction — and to disable the negotiation of specific codecs ([Firefox bug 1396922](https://bugzil.la/1396922)).
```

File: files/en-us/mozilla/firefox/releases/128/index.md [hunk 2]
```diff
@@ -45,12 +45,15 @@ This article provides information about the changes in Firefox 128 that affect d
 
 ### APIs
 
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/releases/128/index.md:49]:
```suggestion
```

**Comment by hamishwillee** [comment]
@Elchi3 When you get a moment, this one should be an easy review.

**Comment by hamishwillee** [comment]
@bsmth Can you please look at this - removed Florian as reviewer because MDN FF work is not a priority for him.

File: files/en-us/mozilla/firefox/releases/128/index.md [hunk 3]
```diff
@@ -51,6 +51,7 @@ This article provides information about the changes in Firefox 128 that affect d
...
 - {{domxref('MediaKeys.getStatusForPolicy()')}} is now supported for checking whether the CDM module, which is used to decrypt DRM protected content, would allow the presentation of encrypted media data for a "hypothetical" key based on specified policy requirements such as the [High-bandwidth Digital Content Protection (HDCP)](https://en.wikipedia.org/wiki/High-bandwidth_Digital_Content_Protection) version supported by the system.
   This provides an application with a simple mechanism to know in advance whether playback at the optimal resolution will be allowed, without having to create a media key session or fetch a real license. ([Firefox bug 1878714](https://bugzil.la/1878714)).
+- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preference order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/128/index.md:54]: Minor reword, what do you think?

```suggestion
- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preferred order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).
```

**Review by bsmth — APPROVED** [review]
LGTM thank you!

**Comment by bsmth** [comment]
BTW this might be interesting to check: https://blog.mozilla.org/webrtc/cross-browser-support-for-choosing-webrtc-codecs/

File: files/en-us/mozilla/firefox/releases/128/index.md (hunk 3 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/128/index.md:54]: Thanks. I'm ambivalent so I'll accept it :-)

File: files/en-us/mozilla/firefox/releases/128/index.md [hunk 4]
```diff
@@ -51,6 +51,7 @@ This article provides information about the changes in Firefox 128 that affect d
...
 - {{domxref('MediaKeys.getStatusForPolicy()')}} is now supported for checking whether the CDM module, which is used to decrypt DRM protected content, would allow the presentation of encrypted media data for a "hypothetical" key based on specified policy requirements such as the [High-bandwidth Digital Content Protection (HDCP)](https://en.wikipedia.org/wiki/High-bandwidth_Digital_Content_Protection) version supported by the system.
   This provides an application with a simple mechanism to know in advance whether playback at the optimal resolution will be allowed, without having to create a media key session or fetch a real license. ([Firefox bug 1878714](https://bugzil.la/1878714)).
+- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preferred order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/releases/128/index.md:54]: @bsmth FYI only The blog link you pointed me to has inspired me to look at this again and make this a little more clear.

```suggestion
- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the codecs that a WebRTC local peer is able to use for decoding received data, in its preferred codec order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).
```

**Pull request merged**